utils/build/docker/nodejs/app.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-if [ ${UDS_WEBLOG:-} = "1" ]; then
-    ./set-uds-transport.sh
```

Comment:
It's not removed, it's moved to a different file. This is because `app.sh` is a hardcoded entry point, but its code changes depending on the image as some of them generate the file. So there is now a new `app.sh` that is always the same and checks for the volume, and calls out to a separate file that is now generated instead of `app.sh`.

Reply:
Not very important, but is it possible to merge `app.sh` and `server.sh` ?

Reply:
In order to merge them, the Dockerfiles would have to append the dynamic code to the existing static file instead of creating a new one. Definitely doable, but I'm not sure it would be much cleaner. I can definitely do that if that's your preference.

Reply:
Definitly no strong opinion here, I let you choose what you prefer.

Reply:
I ended up moving everything to `app.sh` and appending only the call out to `node`. It's actually cleaner and doesn't require a new file 👍